clang/lib/Frontend/InitPreprocessor.cpp

```diff
@@ -465,6 +465,9 @@ static void InitializeStandardPredefinedMacros(const TargetInfo &TI,
     // SYCL Version is set to a value when building SYCL applications
     if (LangOpts.SYCLVersion == 2017)
       Builder.defineMacro("CL_SYCL_LANGUAGE_VERSION", "121");
+    else if (LangOpts.SYCLVersion == 2020)
+      Builder.defineMacro("CL_SYCL_LANGUAGE_VERSION", "202001");
+
```

Comment:
I meant something like this:
```suggestion
    if (LangOpts.SYCLVersion == 2017) {
      Builder.defineMacro("CL_SYCL_LANGUAGE_VERSION", "121");
      Builder.defineMacro("SYCL_LANGUAGE_VERSION", "201707");
    }
    else if (LangOpts.SYCLVersion == 2020)
      Builder.defineMacro("SYCL_LANGUAGE_VERSION", "202001");

```

Reply:
Sorry to have missed that.  Got it now.  Also updated places were we use old macro to now start to switch to new one.